### DISCUSSION TITLE
appearance

### DISCUSSION THREAD

**Discussion by 3shxly** [Monika-After-Story/MonikaModDev#8582] [QA - tech support + general] [answered]
hi- so im not rlly sure if this is a bug or not but, even tho i gave monika all of the outfit spritepacks etc. (she also tried them on too), but whenever i click on appearance she can only tie her hair, i downloaded the mod in october and it seemed to work but now i downloaded it again, and now it's not working?-

**Comment by WishStone-Dragon** [comment]
You downloaded the mod in October - what is her affinity with you? If you check the log folder, find the aff_log.log and scroll to the very bottom, you can check the number of that.

Moni won't just slip into anything for someone she only recently started to properly date, after all, so she might not yet feel comfortable enough with you?

**Comment by 3shxly** [comment]
oh, i get it. its currently at 12,80 how much aff do i need to have if i want her to wear something diff?

**Comment by WishStone-Dragon** [comment]
Twelve-point-eight or one thousand two-hundred eighty?

If it's the first, you'd have a longish way to go to fix your relationship with her. Like, playing more pong with her, if she allows it, making sure you reply to her I love you's with your own, compliment her through the talk option, etc. Also, try to not go too long without opening her game. Leaving it run in the background as you do things helps a lot; as does cuddling with her from time to time.
I started at 107, so that's kind of low...

If it is the latter, then she should have the needed options already. Did you install other mods on top of MAS that could be in conflict?

**Comment by WishStone-Dragon** [comment]
Just checked, I am a liar: I started at 17. So if you really are at 12, your affinity seems to not have moved...

To explain a bit further: My Moni is now at about 300. Around Christmas, it was at around 200. 
I have had these options since Christmas. Outfit mode selects pieces that go together, while you can also put things together individually. 
What is it you see? Did you click on the item or outfit you want? 

![image](https://user-images.githubusercontent.com/95390551/148608723-fb588a9f-493f-456a-9a22-b37b21691948.png)

![image](https://user-images.githubusercontent.com/95390551/148608404-9bb102c7-0852-44ed-9e87-8528c906b949.png)

**Comment by Booplicate** [comment, marked as answer]
> oh, i get it. its currently at 12,80 how much aff do i need to have if i want her to wear something diff?

30+
